pywbem/_statistics.py

```diff
@@ -45,7 +46,95 @@
 section for the operation names).
 The :class:`~pywbem.OperationStatistic` objects are under control of the
 :class:`~pywbem.Statistics` class.
-"""
+
+The statistics support maintains two kinds of times for each kind of WBEM
+operation:
+
+* Client times: The elapsed times for the WBEMConnection operation methods from
+  call to return. This is measured in pywbem pretty close to the API the pywbem
+  user is calling.
+
+* Server times: The elapsed times for processing the WBEM operations in the
+  WBEM server from receiving the CIM-XML request message to sending back the
+  CIM-XML response message. The server times are not measured by pywbem, but
+  are taken from the `WBEMServerResponseTime` HTTP header field of a CIM-XML
+  response, if present. See :term:`DSP0200` for a description of this header
+  field.
+
+  The `WBEMServerResponseTime` HTTP header field is optional to be implemented
```

Comment:
Change "optional to be implemented" to "optionally implemented"

Reply:
DONE